reviewrot/basereview.py

```diff
@@ -12,6 +13,13 @@
 LastComment = namedtuple('LastComment', ('author', 'body', 'created_at'))
 
 
+def gravatar(email):
+    """ Return the url to the public gravatar for an email. """
+    digest = hashlib.md5(email.strip().lower()).hexdigest()
```

Comment:
This fails on Python 3, hashlib.md5 expects bytes but is getting Unicode string:

`TypeError: Unicode-objects must be encoded before hashing`

Reply:
Ah, of course.  Will encode.  (You're up late!)